socorro/unittest/processor/rules/memory_reports/bad_units.json

```diff
@@ -0,0 +1,15 @@
+{
+ "version": 1,
+ "hasMozMallocUsableSize": true,
+ "reports": [
+  {
+   "process": "Main Process (pid 11620)",
+   "path": "explicit/foo",
+   "kind": 2,
```

Comment:
To clarify, if that's correct, then we should fix the kind so that this test case only has bad units.